signbank/settings/base.py

```diff
@@ -243,6 +243,8 @@
 else:
     GLOSS_VIDEO_FILE_STORAGE = 'signbank.video.models.GlossVideoStorage'
 
+NZSL_SHARE_HOSTNAME = os.getenv('NZSL_SHARE_HOSTNAME')
```

Comment:
Reminder for myself to look up where this new environment variable actually needs to be added for UAT and Prod